tests/run/i3018.flags

```diff
@@ -0,0 +1 @@
+-optimise
```

Comment:
flags files are ignored by vulpix, flags have to be put in the code in CompilationTests.scala

Reply:
fixed in #3029